procedures/mobile_automation_setup.rst

```diff
@@ -32,100 +26,62 @@ administrating them in the live Taskcluster website.
-
-    1. ``HookId`` should be ``$product-nightly``
-    2. ``Name`` should not end with "(staging)"
-    3. Copy the task template from the staging hook, with some
-       modifications
-
-       1. Remove the ``--staging`` parameter from ``payload.command``
-       2. Update ``metadata`` to remove anything mentioning "staging"
-       3. Update the scope to assume from the non-staging hook's role
-
-7.  Create the role for the staging hook (e.g.:
-    ```fenix-nightly-staging`` <https://taskcluster-web.netlify.com/auth/roles/hook-id%3Aproject-mobile%2Ffenix-nightly-staging>`__)
-
-    1. Go to `Roles <https://taskcluster-web.netlify.com/auth/roles>`__
-    2. ``RoleId`` should be idential to your staging hook's ``HookId``,
-       but prefixed with ``hook-id:`` (e.g.:
-       ``hook-id:project-mobile/fenix-nightly-staging``)
-    3. ``Description`` should be something like "Hook for building,
-       signing and uploading (staging) Nightly versions of $product"
-    4. Add all the scopes you'll need to fulfill the requirements in you
-       ``.taskcluster.yml`` file (when ``is_mozilla_mobile_repo`` is
-       ``false``)
-
-8.  Do the same thing, but for non-staging nightlies (e.g.:
-    ```fenix-nightly`` <https://taskcluster-web.netlify.com/auth/roles/hook-id%3Aproject-mobile%2Ffenix-nightly>`__)
-
-    1. Remove mentions of "staging" from each field
-    2. Make sure that all scopes are provided to fulfill requirements in
-       ``.taskcluster.yml`` (when ``is_mozilla_mobile_repo`` is
-       ``true``)
+3.  Add ``.taskcluster.yml`` in the root of the repository. This file
+    tells Taskcluster what to do upon github events happening (push,
+    pr, release, etc). Since we're going to want to run ``taskgraph``
+    to decide what tasks to run, we can take a ``.taskcluster.yml`` from
+    a similarly-configured repository, like `fenix
+    example <https://github.com/mozilla-mobile/fenix/blob/master/.taskcluster.yml>`__)
+
+    -  Update repository and treeherder references to refer to your project,
+       rather than ``fenix``.
+
+4.  Implement ``taskgraph`` configuration for the repository. See the
+    `Fenix configuration <https://github.com/mozilla-mobile/fenix/tree/master/taskcluster>`__.
+    You'll need to implement the following parts:
+
+    -  Define tasks in YAML in ``taskcluster/ci/``
+    -  Define transforms in ``taskcluster/$project_taskgraph/transforms/`` which operate
+       on the tasks defined in the YAML
+    -  Define any custom loaders in ``taskcluster/$project_taskgraph/loader/`` (this is
+       useful in cases like needing to generate a dynamic number of tasks based on an
+       external source, like ``gradle`` or a ``.buildconfig.yml`` file)
+    -  Define ``Dockerfiles`` in ``taskcluster/docker/``
+
+5.  If you have schedule-based automation, create cron hook
+    (copy from the `fenix hook <https://tools.taskcluster.net/hooks/project-releng/cron-task-mozilla-mobile-fenix>`__)
```

Comment:
Nit: No need 😃  `ci-admin` generates it for you

Reply:
How do you generate the cron hook with `ci-admin`? I can add it to the docs here

Reply:
Thanks to these 2 lines (if I remember correctly):

* https://hg.mozilla.org/ci/ci-configuration/file/6e2e5169451be654cbb8a6962fca9327250d57a3/projects.yml#l483
* https://hg.mozilla.org/ci/ci-configuration/file/6e2e5169451be654cbb8a6962fca9327250d57a3/projects.yml#l485

Reply:
Thanks!